### PR TITLE
Some lowercase

### PR DESCRIPTION
**Pull request by matentzn**
This was a 5 min job with search replace... Gives us a headstart

### CODE REVIEW

**Comment by matentzn** [comment]
Second attempt :D

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -302347,7 +302347,7 @@ consider: UMLS:C0014175 {source="Orphanet:137820"}
 [Term]
 id: MONDO:0015306
 name: Lemierre syndrome
-def: "and that is characterized by septic thrombophlebitis of the internal jugular vein that leads to septic, usually pulmonary, embolism, associated with ENT (ear, nose, and throat) infection that manifests with fever, neck pain, and tonsillopharyngitis." [Orphanet:137839]
+def: "And that is characterized by septic thrombophlebitis of the internal jugular vein that leads to septic, usually pulmonary, embolism, associated with ENT (ear, nose, and throat) infection that manifests with fever, neck pain, and tonsillopharyngitis." [Orphanet:137839]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:302350]: this one seems like it is truncated

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:302350]: The def should be:

Lemierre syndrome is a rare, potentially lethal, oropharyngeal infectious disease occurring in immunocompetent adolescents and young adults that is mainly due to Fusobacterium necrophorum and that is characterized by septic thrombophlebitis of the internal jugular vein that leads to septic, usually pulmonary, embolism, associated with ENT (ear, nose, and throat) infection that manifests with fever, neck pain, and tonsillopharyngitis.

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -362844,7 +362844,7 @@ property_value: seeAlso https://rarediseases.info.nih.gov/diseases/10353/porphyr
...
 id: MONDO:0019143
 name: angiostrongyliasis
-def: "and that is acquired by the ingestion of the infective larvae on vegetables or in raw or undercooked snails, slugs, land crabs, freshwater shrimps, frogs and lizards. The main feature is eosinophilic meningitis, with clinical manifestations including fever, headache, malaise, fatigue, vomiting, rhinorrhea, blurred vision, diplopia, cough, stiff neck, enteritis, constipation and paraesthesia due to the movement of the worms from the intestines to the lungs, central nervous system and eyes. In severe cases without treatment, coma and death can occur." [Orphanet:74]
+def: "And that is acquired by the ingestion of the infective larvae on vegetables or in raw or undercooked snails, slugs, land crabs, freshwater shrimps, frogs and lizards. The main feature is eosinophilic meningitis, with clinical manifestations including fever, headache, malaise, fatigue, vomiting, rhinorrhea, blurred vision, diplopia, cough, stiff neck, enteritis, constipation and paraesthesia due to the movement of the worms from the intestines to the lungs, central nervous system and eyes. In severe cases without treatment, coma and death can occur." [Orphanet:74]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:362847]: The def should be:

A foodborne zoonotic disease, endemic to Southeast Asia and the Pacific Islands, caused by the rat lungworm Angiostrongylus cantonensis and that is acquired by the ingestion of the infective larvae on vegetables or in raw or undercooked snails, slugs, land crabs, freshwater shrimps, frogs and lizards. The main feature is eosinophilic meningitis, with clinical manifestations including fever, headache, malaise, fatigue, vomiting, rhinorrhea, blurred vision, diplopia, cough, stiff neck, enteritis, constipation and paraesthesia due to the movement of the worms from the intestines to the lungs, central nervous system and eyes. In severe cases without treatment, coma and death can occur.

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]

**Comment by matentzn** [comment]
Now?

**Review by nicolevasilevsky — APPROVED** [review]
Looks great!

It's so satisfying to have this fixed. :)

**Pull request merged**